Algolia.Search/Index.cs

```diff
@@ -590,9 +590,13 @@ public Task<JObject> BrowseFromAsync(Query q, string cursor, RequestOptions requ
             string cursorParam = "";
             if (cursor != null && cursor.Length > 0)
             {
-                cursorParam = string.Format("&cursor={0}", WebUtility.UrlEncode(cursor));
+                Dictionary<string, object> body = new Dictionary<string, object>();

```

Comment:
It's a nit but we could use a JObject instead of a dictionary 
`JObject body = new JObject {"cursor", cursor };`